index.html

```diff
@@ -0,0 +1,250 @@
+                <li><p>The giants of the web assemble.Europe’s biggest tech conference.</p></li>
+            </ul>
+            <ul class="bg-program">
+                <li><img src="./images/programicon05.png" alt="Exhibition"></li>
+                <li><h3>Reason to Join</h3></li>
+                <li><p>Three reasons: incredible speakers, unparalleled networking opportunities, and proprietary software that will maximise your experience.</p></li>
+            </ul>
+            <button class="join-btn"><a class="buttonlink" href="#">Register as Tech</a></button> 
+        
+        </div>
+    </div>
+    <div class="cover-program-desktop">
+        <div class="program-list-desktop">
+            <h2>Main Program<hr class="hr"></h2>
+            <div class="desktop-wrapper">
+                <ul class="bg-program-desktop">
+                    <li><img src="./images/programicon1.png" alt="Networking"></li>
+                    <li><h3>Networking</h3></li>
+                    <li> <p>CGTN has said that “Web Summit brings together the people and companies redefining the global tech industry”.</p></li>
+                </ul>
+            <ul class="bg-program-desktop">
+                <li><img src="./images/programicon2.png" alt="lead"></li>
+                <li><h3>Lead generation</h3></li>
+                <li><p>We live in uncertain times for tech and business. At Web Summit, we welcome the people who are tackling the big challenges facing industry today.</p></li>
+            </ul>
+            <ul class="bg-program-desktop">
+                <li><img src="./images/programicon3.png" alt="learning"></li>
+                <li><h3>Learning and development</h3></li>
+                <li><p>Web Summit’s educational content, informative masterclasses and innovative roundtables mean that you can bring valuable skills back to your office.</p></li>
+            </ul>
+            <ul class="bg-program-desktop">
+                <li><img src="./images/programicon4.png" alt="Journal"></li>
+                <li><h3>Wall Street Journal</h3></li>
+                <li><p>The giants of the web assemble.Europe’s biggest tech conference.</p></li>
+            </ul>
+            <ul class="bg-program-desktop">
+                <li><img src="./images/programicon05.png" alt="Exhibition"></li>
+                <li><h3>Reason to Join</h3></li>
+                <li><p>Three reasons: incredible speakers, unparalleled networking opportunities, and proprietary software that will maximise your experience.</p></li>
+            </ul>
+        </div>
+        <h5 class="whole-link"><a>SEE THE WHOLE PROGRAM</a></h5>
+        </div>
+    </div>
+</section>
+<article id="featured-mobile">
+    <h3 class="featuredh">Featured Speakers</h3>
+    <hr class="hr1"/>
+    <section class="featured-speaker">
+  <div class="image-of-background">
+<img class="featured-speakers" src="./images/speaker1.jpg" alt="speaker1">
+  </div>
```

Comment:
- [ OPTIONAL]  After clicking see More button,  the alignment of featured speaks is getting destroyed. consider the alignments of the speakers to be well organized as it is from the design. 
![image](https://user-images.githubusercontent.com/43877689/219643843-22ad9d1f-5393-40d1-a9bd-509ba430685b.png)

Reply:
All the optional changes I will incorporate them later on. Please consider the time factor I also have coding challenges to complete for this week. This is my fourth review that I am sending again for a review please approve

Reply:
This optional change has also been incorporated. screen shot attached for your kind reference. 
![image](https://user-images.githubusercontent.com/117091923/221371156-6e433fa2-0b5c-4ed8-b204-3518874d1084.png)